test/perl/api-gateway/validation/oauth2/oauthTokenValidator.t

```diff
@@ -130,21 +132,26 @@ env REDIS_PASS_OAUTH;
             set $authtoken $http_authorization;
             set_if_empty $authtoken $arg_user_token;
             set_by_lua $authtoken 'return ngx.re.gsub(ngx.arg[1], "bearer ", "","ijo") ' $authtoken;
-            set_md5 $authtoken_hash $authtoken;
 
-            set $key 'cachedoauth:$authtoken_hash';
             content_by_lua '
+                local hasher = require "api-gateway.util.hasher"
+                local oauthTokenHash = ngx.var.authtoken_hash
```

Comment:
Is this line necessary?

Reply:
It is used to form the cached key for Redis 
```key = "cachedoauth:" .. oauthTokenHash```

Reply:
Actually, it's not needed; you have 2 assignments for local variable oauthTokenHash :
                local oauthTokenHash = ngx.var.authtoken_hash
                ...
                oauthTokenHash = hasher.hash(ngx.var.authtoken)